src/common/snippets/src/lowered/port_descriptor.cpp

```diff
@@ -149,8 +149,8 @@ void set_port_desc(const T& port, std::vector<size_t> subtensor, std::vector<siz
 void PortDescriptorUtils::set_port_descriptor(const ov::Input<ov::Node>& in, std::vector<size_t> subtensor, std::vector<size_t> layout) {
     set_port_desc(in, subtensor, layout);
 }
-void PortDescriptorUtils::set_port_descriptor(const ov::Output<ov::Node>& in, std::vector<size_t> subtensor, std::vector<size_t> layout) {
-    set_port_desc(in, subtensor, layout);
+void PortDescriptorUtils::set_port_descriptor(const ov::Output<ov::Node>& out, std::vector<size_t> subtensor, std::vector<size_t> layout) {
+    set_port_desc(out, subtensor, layout);
```

Comment:
Minor: Here and on L150. Please use std::move or pass by const ref to avoid redundant copies.
```suggestion
    set_port_desc(out, std::move(subtensor), std::move(layout));
```